composer.json

```diff
@@ -31,12 +31,12 @@
 		"php": ">=5.6.0"
 	},
 	"require-dev": {
-		"phpcompatibility/phpcompatibility-wp": "^2.1.0",
+		"phpcompatibility/phpcompatibility-wp": "^2.1.3",
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
-		"squizlabs/php_codesniffer" : "^3.5.4",
-		"wp-coding-standards/wpcs": "*",
+		"squizlabs/php_codesniffer" : "^3.6.0",
+		"wp-coding-standards/wpcs": "~2.3.0",
 		"wp-phpunit/wp-phpunit": "^5.9",
-		"phpunit/phpunit": "^7.5",
+		"phpunit/phpunit": "^9.0",
 		"yoast/phpunit-polyfills": "^1.0.1"
 	}
```

Comment:
Could we also add support to composer v2?

```
"config": {
	"allow-plugins": {
		"composer/installers": true,
		"dealerdirect/phpcodesniffer-composer-installer": true
	}
}
```

Reply:
Sure, I'll add it 👍